examples/active-active/README.md

```diff
@@ -0,0 +1,73 @@
+
+## About This Example
+
+This example for Terraform Enterprise creates a TFE installation with the following traits.
+
+-  Active-Active
+-  External Services
+-  Non-Airgap
+-  m5.xlarge
+-  RHEL 7.9
+-  A privately accessible TCP load balancer with TLS pass-through
+-  An ubuntu based mitm proxy server with TLS termination 
+
+## Prerequisites
+
+This example assumes that the following resources exist:
+
+- TFE license is on a filepath defined by var.license_file
+- A DNS zone
+- Valid managed SSL certificate to use with load balancer:
+  - Create/Import a managed SSL Certificate using AWS ACM to serve as the certificate for the DNS A Record
+- Valid CA certificate and private key for MITM proxy:
+  - AWS Secret Manager Secret name which contains the Base64 encoded version of a PEM encoded certificate and private key for mitm proxy server
+
+## How to Use This Module
+
+### Deployment
+
+ 1. Read the entire README.md for the root module.
+ 2. Ensure account meets module pre-requisites from above.
+ 3. Clone repository.
+ 4. Change directory into desired example folder.
+ 5. Create a local terraform.auto.tfvars file and instantiate the required inputs as required in respective ./examples/active-active/variables.tf including the path to the license under the license_file variable value.
+ 6. Authenticate against aws provider. See [instructions](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration).
+ 7. Initialize terraform and apply the module configurations using below commands.
+
+    NOTE: `terraform plan` will print out the execution plan which describes the actions Terraform will take in order to build your infrastructure to match the module configuration. If anything in the plan seems incorrect or dangerous, it is safe to abort here and do not proceed to `terraform apply`
+
+    ```
+    terraform init
+    terraform plan
+    terraform apply
+    ```
+
+## Post-deployment Tasks
+
+The build should take approximately 10-15 minutes to deploy. Once the module has completed, give the platform another 10 minutes or so prior to attempting to interact with it in order for all containers to start up.
+
+Unless amended, this example will not create an initial admin user using the IACT, but does output the URL for convenience. Follow the advice in this document in order to create the initial admin user, and login to the system using this user in order to configure it for use.
+
+### Connecting to proxy server
+
+1. To create a tunnel for Chrome:
+   By default, Chrome uses your macOS or Windows proxy. To change your proxy settings from within Chrome, take the following steps: 
+   - Open the Chrome toolbar and select Settings.
+   - Scroll down to the bottom of the display. Click on Show advanced settings...
+   - Scroll down to “System” and choose Open your computer’s proxy settings.
+   - Set Chrome proxy server settings.
```

Comment:
```suggestion
   - Open the Chrome toolbar and select "Settings".
   - Scroll down to the bottom of the display. Click on "Show advanced settings".
   - Scroll down to “System” and choose "Open your computer’s proxy settings".
   - Set Chrome proxy server settings.
```

I only corrected grammar there, but actually, these are not the instructions that I have followed. Curious why you didn't use these:

- Applications > System Preferences > Network > Advanced > Proxies 
- [X] SOCKS Proxy
- localhost : 5000
- OK —> Apply

Reply:
That's only applicable for MAC systems and different for windows